projects/helper/tokenMapping.js

```diff
@@ -72,6 +72,9 @@ const fixBalancesTokens = {
     'FX': { coingeckoId: "fx-coin", decimals: 18 },
     'usdt': { coingeckoId: "tether", decimals: 6 },
   },
+  xpla: {
+    'axpla': { coingeckoId: "xpla", decimals: 18 },
```

Comment:
hi @park1847 , I have refactored your code a bit, can you add other xpla tokens on coingecko here?

Reply:
or if you dont want to add more, I can merge this PR as is

Reply:
There's nothing more to add :)